tr55/makeMiniAppTable.py

```diff
@@ -44,9 +43,6 @@
 
     soil_types = ['a', 'b', 'c', 'd']
 
-    et_max = 0.207
```

Comment:
Looks like this constant was moved internally, which seems good.  What about `pre_columbian`, is this still specifiable despite being removed here?